install.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+cd /vagrant
+
+gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
+curl -L https://get.rvm.io | bash -s stable
+
+source $HOME/.rvm/scripts/rvm
```

Comment:
Perhaps consider echoing this into `~vagrant/.bash_profile` so that people using the image don't have to manually do that step of the rvm setup process.